lib/graphql/github.graphql

```diff
@@ -1018,3 +1062,31 @@ query Issue($owner: String!, $name: String!, $number: Int!, $cursor: String) {
     }
   }
 }
+
+# Releases
+query Releases($name: String!, $owner: String!, $cursor: String) {
+   repository(name: $name, owner: $owner) {
+    releases(first: 30, after: $cursor) {
```

Comment:
Could we set the order? DESC by time would be fine, just the same as the web releases page